pingdom-checks/veteran-verification.ping

```diff
@@ -25,3 +26,29 @@ pingdom save-check \
   -a group=veteran-verification \
   -a userids_csv="$STATUSPAGE_SANDBOX_VETERAN_VERIFICATION" \
   -a integrationids_csv="$VETERAN_VERIFICATION_SLACK_ID,$PAGER_DUTY"
+
+#sandbox
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-veteran-verification-health \
+  -a host=sandbox-api.va.gov \
+  -a url="/services/veteran_verification/v2/healthcheck/liveness" \
+  -a port="443" \
+  -a responsetime_threshold="30000" \
+  -a resolution="1" \
+  -a group=veteran-verification \
+  -a userids_csv="$NO_USERS" \
+  -a integrationids_csv="$VETERAN_VERIFICATION_SLACK_ID,$PAGER_DUTY"
+
+#sandbox
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-veteran-verification-health \
```

Comment:
This should be renamed to sandbox-veteran-verification-v2-backend-health, since it's the readiness probe.